config.json

```diff
@@ -739,7 +745,7 @@
         "name": "Grains",
         "uuid": "d003975a-9045-4f03-9ad9-c15db584dc13",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["conditionals", "errors", "numbers", "math"],
```

Comment:
Here is my solution : https://exercism.org/tracks/javascript/exercises/grains/solutions/TomPradat

I'm not sure about errors, I added it because you need to throw errors in the exercise. Though that's the case of many practice exercises, and the errors concept is very advanced in the hierarchy, what do you think ?

Reply:
It's the right choice because you must understand how errors work. We can re-evaluate later.